module-2/middlewares/check-jwt.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response, NextFunction } from 'express';
+import AuthService from '../services/AuthService';
+
+export default function (req: Request, res: Response, next: NextFunction): void {
+  const auth = req.headers.authorization;
+
+  if (!auth) {
+    res.status(401).send({ message: 'Not authorized' });
+  } else {
```

Comment:
"Else" part is redundant in term of a typical code style for Node.js controllers.
`if (!auth)` part looks like an "early return" pattern.

Reply:
Fixed